projects/ngx-sub-form/src/lib/ngx-sub-form.component.ts

```diff
@@ -50,12 +24,15 @@ export abstract class NgxSubFormComponent<ControlInterface, FormInterface = Cont
   @Input()
   public formControlName: string | undefined = undefined;
 
+  // can't define them directly
+  protected abstract getFormControls(): Controls<FormInterface>;
```

Comment:
This is a breaking change; commit should say this

Reply:
:zipper_mouth_face: well spotted!

Reply:
Done